applications/fishing-map/src/features/map/map.slice.ts

```diff
@@ -95,73 +95,78 @@ type SublayerVessels = {
   vessels: ExtendedFeatureVessel[]
 }
 
+const getInteractionEndpointDatasetConfig = (getState: () => any, features: ExtendedFeature[]) => {
+  const state = getState() as RootState
```

Comment:
In this case where only the temporalgridDataviews param is need, for me it is easier to have just the param instead of the function which returns the state... so I replaced the getState with:
```js
const getInteractionEndpointDatasetConfig = (
  features: ExtendedFeature[],
  temporalgridDataviews: UrlDataviewInstance[] = []
)
```
also to take advantage of typings